tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
you've got `dev-scripts` in devDeps, so you've got dev-config and all our extendable tsconfigs.  

Any of those work?